### PR TITLE
Add operation ID to headers

### PR DESCRIPTION
**Pull request by bessbd**
## PR summary

Fixes: #10 

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java [hunk 1]
```diff
@@ -39,4 +40,11 @@ public void testGetSdkHeaders() {
         String expectedUserAgentRegex = String.format("%s/%s .*", SdkCommon.getProjectName(), SdkCommon.getVersion());
         assertTrue(userAgent.matches(expectedUserAgentRegex));
     }
+
+    @Test
+    public void testAnalyticsHeaders() {
+        Map<String, String> headers = SdkCommon.getSdkHeaders("service1", "v1", "operation1");
+        String userAgent = headers.get("X-IBMCloud-SDK-Analytics");
```
Comment (ricellis) [inline modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java:47]: nit: rename the var, it's a little confusing calling this `userAgent`

**Review by ricellis — APPROVED** [review]
+1
1 nit that I think would beneficial to address

File: modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java (hunk 1 above)
Comment (bessbd) [reply modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java]: Done in ce72d3d

**Review by emlaver — APPROVED** [review]

**Comment by bessbd** [comment]
Thank you for the review, @ricellis and @emlaver . I'm about to merge this.

**Pull request merged**